convert2rhel/actions/conversion/preserve_only_rhel_kernel.py

```diff
@@ -143,22 +143,29 @@ def run(self):
             None,
         )
         if kernel_to_change:
-            description = "Detected leftover boot kernel, changing to RHEL kernel"
-            loggerinst.warning(description)
+            diagnosis = "Detected default boot kernel {} in /etc/sysconfig/kernel.".format(kernel_to_change)
+            # need to change to "kernel" in rhel7 and "kernel-core" in rhel8
+            new_kernel_str = "DEFAULTKERNEL=" + ("kernel" if system_info.version.major == 7 else "kernel-core")
+            loggerinst.warning(diagnosis)
             self.add_message(
                 level="WARNING",
-                id="LEFTOVER_BOOT_KERNEL_DETECTED",
-                title="Leftover boot kernel detected",
-                description=description,
+                id="LEFTOVER_DEFAULT_BOOT_KERNEL_DETECTED",
```

Comment:
We shouldn't change the ID. Rest is fine. This to me counts as a breaking change otherwise.
```suggestion
                id="LEFTOVER_BOOT_KERNEL_DETECTED",
```

Reply:
You're right. I can create a separate ticket just for changing the message IDs and target it for 3.0.0.

Reply:
https://github.com/oamg/convert2rhel/pull/1447